### ISSUE TITLE
Presence are not sent to the client

### ISSUE THREAD

**Issue by open-sudo** [Auties00/Cobalt#302]
I send presence to client like this:
```
 whatsapp.changePresence(info.chat(), ContactStatus.COMPOSING);
```
Unfortunately client never sees it. Thoughts?

**Comment by Auties00** [comment]
I'll look into it

**Comment by Auties00** [comment]
Can you test the master branch? It should be fixed now

**Comment by open-sudo** [comment]
Thanks for the fix. The good is that there is no crash anymore when presence is sent. However, I do not see the composing status. materialized in the client. I was expecting to see "Typing..." as status.

**Comment by Auties00** [comment]
> Thanks for the fix. The good is that there is no crash anymore when presence is sent. However, I do not see the composing status. materialized in the client. I was expecting to see "Typing..." as status.

Does this happen in both private chats and groups?

**Comment by open-sudo** [comment]
I tested only in private chats.

**Comment by Auties00** [comment]
fixed in the latest commit, will be available on 3.4.10
